src/main/java/com/thealgorithms/bitmanipulation/HighestSetBit.java

```diff
@@ -0,0 +1,26 @@
+package com.thealgorithms.bitmanipulation;
+
+/**
+ * Find Highest Set Bit
+ * @author Bama Charan Chhandogi (https://github.com/BamaCharanChhandogi)
+ */
+
+public class HighestSetBit {
```

Comment:
```suggestion
public final class HighestSetBit {
    private HighestSetBit() {
    }
```

Please note that there are two things here:
- having the _default_ constructor marked as `private` makes it impossible to have objects of that class (it does not make sense to have any object of that class),
- making this class as `final` makes it impossible to derive new ones from of (again: it does not make sense to do it).

Above makes that class a _proper utility class_.